### PR TITLE
[GLOBAL/SCSS] Add Font-size variables

### PR DESCRIPTION
**Pull request by tureey**
# Details
## Description
Add font-size variable so everyone needs to use only this sizes.

### CODE REVIEW

File: src/theme/base/_variables.scss [hunk 1]
```diff
@@ -1,4 +1,13 @@
 $font-main: 'Open Sans', sans-serif;
-
+// Palette
 $color-main: #EB6E64;
 $color-white: #FFF;
+// Font-sizes
+$fz-xxs: 0.75rem; // 12px
```
[the remark this answers is not in the document]
Comment (tureey) [reply src/theme/base/_variables.scss]: Yes,  I am used to work with the `Emmet abbreviations` for all the `css properties`.

File: src/theme/base/_variables.scss [hunk 2]
```diff
@@ -1,4 +1,13 @@
...
 $color-white: #FFF;
+// Font-sizes
+$font-xxs: 0.75rem; // 12px
+$font-xs: 0.875rem; // 14px
+$font-s: 1rem; // 16px
+$font-m: 1.125rem; // 18px
+$font-l: 1.5rem; // 24px
+$font-xl: 2rem; // 32px
+$font-xxl: 4.25rem; // 68px
+$font-giant: 4.25rem; // 84px
```
Comment (arthurbuhl) [inline src/theme/base/_variables.scss]: I think you need to update the size here like:

```diff
- $font-giant: 4.25rem; // 84px
+ $font-giant: 5.25rem; // 84px
```

Comment (tureey) [reply src/theme/base/_variables.scss]: duude.. really! Happened the same in the old PR! Good eye mate!

Good job

**Comment by tureey** [comment]
@crysfel who can merge? I thought I would have permission so, your turn 😛

**Comment by tureey** [comment]
> I can merge, but we need a second approval.

It's weird because, in the feed I don't see the second approval but, In the resumé on the top, it says 2 approvals already.

**Comment by emmawedekind** [comment]
image

**Comment by arecvlohe** [comment]
@tureey 

> It's weird because, in the feed I don't see the second approval but, In the summary on the top, it says 2 approvals already.

2 approvals from those with _write_ access

**Comment by palashmon** [comment]
Please fix the conflicts here, so that we can merge this PR asap.

**Comment by tureey** [comment]
I will try to do it in 8hours or so. I am going to work and after that I will be traveling. I will do it meanwhile I travel

**Comment by crysfel** [comment]
I will merge as soon as the conflicts are resolved :)

**Comment by arthurbuhl** [comment]
If needed I can create a new PR for this by copying the code from here. I can also resolve any conflicts if present asap and we can merge it. :rocket:

**Comment by arecvlohe** [comment]
@arthurbuhl Can you go into @tureey's fork and push a commit that resolves the conflicts on this branch?

**Comment by crysfel** [comment]
> If needed I can create a new PR for this by copying the code from here. I can also resolve any conflicts if present asap and we can merge it. 🚀

Yes please, there are a couple of open PRs that need these variables. Ping me in slack when ready, I will merge it right away!

**Comment by arthurbuhl** [comment]
> 
> 
> @arthurbuhl Can you go into @tureey's fork and push a commit that resolves the conflicts on this branch?

I tired but it says

> You’re editing a file in a project you don’t have write access to. Submitting a change to this file will write it to a new branch in your fork arthurbuhl/coding-coach, so you can send a pull request.

**Comment by tureey** [comment]
I n 5h It will be done. I am still at google offices. I'll do it in the train

**Comment by tureey** [comment]
This PR is now deprecated and should be closed.
Am I right? @emmawedekind @crysfel

**Confirm and I'll close it**

**Comment by arthurbuhl** [comment]
> 
> 
> This PR is now deprecated and should be closed.
> Am I right? @emmawedekind @crysfel
> 
> **Confirm and I'll close it**

Its been already closed by Crysfel 20 hours ago. Is it still showing open for you?

**Comment by tureey** [comment]
Well, the branch is not removed. I usually merge and close branch! @arthurbuhl

**Comment by emmawedekind** [comment]
Yes, let's delete the branch please :)

**Pull request closed without merging**